packages/renderer/src/lib/kube/KubernetesDashboardGuideCard.svelte

```diff
@@ -0,0 +1,16 @@
+<script lang="ts">
+import { Button } from '@podman-desktop/ui-svelte';
+
+let {
+  title,
+  link,
+}: {
+  title: string;
+  link: string;
+} = $props();
+</script>
+
+<div class="flex flex-col gap-4 p-4 bg-[var(--pd-invert-content-card-bg)] rounded-md items-center">
+  <div class="font-bold text-[var(--pd-invert-content-card-text)]">{title}</div>
+  <Button onclick={() => window.openExternal(link)}>Read more</Button>
```

Comment:
todo: use a function (I gave the reason upper)

Reply:
Fixed.